terraform/environment/main.tf

```diff
@@ -9,15 +20,16 @@ module "eu-west-1" {
   pagerduty_token          = var.pagerduty_token
   management_role          = var.management_role
   default_role             = var.default_role
-
+  lambda_repository_url    = data.aws_ecr_repository.performance_platform_worker.repository_url
+  lambda_ecr_arn           = data.aws_ecr_repository.performance_platform_worker.arn
   providers = {
     aws            = aws.eu_west_1
     aws.management = aws.management
   }
 }
 
 module "eu-west-2" {
-  count                    = local.dr_enabled ? 1 : 0
+  count                    = local.account.dr_enabled && local.account_name == "development" ? 1 : 0
```

Comment:
Is this not needed for the eu-west-1 config?

Reply:
Nope - eu-west-1 will be on always at present. this might change later for failback, but the strategy has a spike in to work out what to do.